src/requestor/routes/manage.py

```diff
@@ -84,56 +84,58 @@ async def create_request(
 
     If no "username" is specified in the request body, will create an access
     request for the user who provided the token.
+
+    The request should include one of the following for which access is being granted:
+
+      * policy
+      * resource_paths + existing role_ids
+      * resource_path[s] without a role_id (a default reader role is assigned)
+
+    "resource_paths" will take precedence over "resource_path" if both are present in the request.
+
     """
     data = body.dict()
     request_id = str(uuid.uuid4())
     logger.info(
         f"Creating request. request_id: {request_id}. Received body: {data}. Revoke: {'revoke' in api_request.query_params}"
     )
 
-    # error (if we have both policy_id and (resource_path or resource_paths))
+    # cast resource_path as list if resource_paths is not present
+    if data.get("resource_path") and not data.get("resource_paths"):
+        data["resource_paths"] = [data["resource_path"]]
+
+    # error (if we have both policy_id and resource_paths)
     # OR (if we have neither)
-    if bool(data.get("policy_id")) == (
-        bool(data.get("resource_paths")) or bool(data.get("resource_path"))
-    ):
-        msg = f"The request must have either resource_paths or a policy_id."
+    if bool(data.get("policy_id")) == bool(data.get("resource_paths")):
+        msg = f"The request must have either resource_path[s] or a policy_id."
         raise_error(logger, msg, body)
```

Comment:
not important, but in case you didn't see it: https://github.com/uc-cdis/requestor/pull/39#discussion_r931496745